airflow/contrib/hooks/ssh_hook.py

```diff
@@ -54,7 +56,8 @@ def __init__(self,
                  username=None,
                  password=None,
                  key_file=None,
-                 timeout=10
+                 timeout=10,
+                 keepalive_interval=30
```

Comment:
nit: perhaps better leave default interval as is (keepalive_interval = 0) so the behavior is consistent with past and with the library's default :)